src/data-interfaces/HoloFuelDnaInterface.js

```
@@ -105,24 +108,51 @@ const presentCheque = ({ origin, event, stateDirection, eventTimestamp, fees, pr
   }
 }
 
-function presentPendingRequest (transaction) {
-  const { event, provenance } = transaction
+const presentDeclinedTransaction = declinedTx => {
+  if (!declinedTx[2]) return console.log('declinedTx[2] is UNDEFINED : ', declinedTx)
```

should throw.

we want to keep `console.log` as clean as possible. Fine for debugging, but in `develop` we almost never want to be calling `console.log`. And especially when something unexpected is happening.

Specifically in this case:

presentCanceledTransaction should always return an object in the shape of our graphql Transaction type. So when you return `undefined` (which is the return value of console.log), graphql will complain and throw an error anyway and it will be harder to debug because the error will be further from the source. 

In general, we want to follow the principle `fail early, fail loudly`

http://oncodingstyle.blogspot.com/2008/10/fail-early-fail-loudly.html